config/crd/bases/kmm.sigs.k8s.io_modules.yaml

```diff
@@ -2077,6 +2077,48 @@ spec:
                               description: Regexp is a regular expression to be match
                                 against node kernels.
                               type: string
+                            sign:
```

Comment:
I guess all of this is related to the fact you are still using your personal image in the PR?!

Reply:
no, i think it is due to the fact that Chris probably did not push the generated files, when he pushed the sign change into CRD